release/api/v1alpha1/release.go

```diff
@@ -26,7 +26,7 @@ type ReleaseSpec struct {
 	// +kubebuilder:validation:Required
 	// +kubebuilder:validation:Minimum=1
 	// Monotonically increasing release number
-	Number int `json:"number,omitempty"`
+	Release string `json:"number,omitempty"`
```

Comment:
```suggestion
	Release string `json:"release,omitempty"`
```

Reply:
But would our release ever be something else other than a number? We shouldn't have to go with the CRD approach for postsubmits right? It is only to inform customers of the locations of the artifacts, but we would already have that configured in our scripts.